scripts/test_real_world_ingestion.py

```
@@ -0,0 +1,885 @@
+        )
+        
+        # Send duplicate
+        response2 = await self.client.post(
+            f"{BASE_URL}/api/v1/ingestion/events",
+            headers=HEADERS,
+            json=event
+        )
+        
+        if response1.status_code == 200 and response2.status_code == 200:
+            # Both succeeded - CHECK IF THIS IS CORRECT BEHAVIOR
+            print(f"[WARN] Both requests succeeded (potential duplicate)")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)